csfieldguide/templates/interactives/matrix-simplifier.html

```diff
@@ -0,0 +1,173 @@
+{% extends interactive_mode_template %}
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/64213422-09551a80-cf01-11e9-9b4a-6ef66e0b45af.png)

A big enough number will push the matrix out of it's container juuust a teeny tiny bit

Reply:
Have made an issue for this #1138